schema.graphql

```diff
@@ -0,0 +1,14 @@
+type Nft @entity(immutable: false) {
+  id: ID!
+  address: String!
+  tokenId: BigInt!
+
+  currentOwner: Account!
+  previousOwner: Account!
+  originalOwner: Account!
```

Comment:
Since we are not following the old subgraph pattern. Maybe doesn't we should only save the current owner

```suggestion
 owner: Account!
```